stat/distmat/unit_vector.go

```diff
@@ -0,0 +1,50 @@
+// Copyright ©2020 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distmat
+
+import (
+	"golang.org/x/exp/rand"
+
+	"gonum.org/v1/gonum/mat"
+	"gonum.org/v1/gonum/stat/distuv"
+)
+
+const unitVecMinLength = 1e-5
```

Comment:
This is a curious name: a unit name has length 1 by definition. Maybe `sampledVecMinLength`?

Reply:
A comment explaining would be helpful, or moving it to immediately near where it's used and calling it `minLength` so that the context explains it.